lms/static/sass/multicourse/_home.scss

```diff
@@ -93,7 +93,7 @@ $course-search-input-height: ($button-size);
         p {
           @extend h2;
           margin-bottom: 0;
-          text-transform: lowercase;
+          text-transform: none;
```

Comment:
This is to keep the legend, "It works! This is the default homepage for this Open edX instance" from being shown in all-lowercase.